core/src/main/java/org/mvcspec/ozark/binding/validate/ConstraintViolations.java

```diff
@@ -83,17 +88,20 @@ else if (lastNode.getKind() == ElementKind.PARAMETER && nodes.size() == 2) {
 
     private static Annotation[] getPropertyAnnotations(ConstraintViolation<?> violation, Path.PropertyNode node) {
 
+        Class<?> leafBeanClass = violation.getLeafBean().getClass();
+        Set<Annotation> allAnnotations = new HashSet<>();
         try {
 
-            Class<?> leafBeanClass = violation.getLeafBean().getClass();
             Field field = leafBeanClass.getDeclaredField(node.getName());
-
-            return field.getAnnotations();
+            allAnnotations.addAll(Arrays.asList(field.getAnnotations()));
 
         } catch (NoSuchFieldException e) {
-            throw new IllegalStateException(e);
+            // ignore for now
```

Comment:
Maybe we should still throw an exception in this case? But I'm not 100% sure. If we cannot find the field we most likely also won't be able to find getters and setters, because in this case that code that extracted the correct `Node` is broken.